sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_models.py

```diff
@@ -315,16 +319,18 @@ def __init__(self, key, secret_uri, label=None, **kwargs):
         self.last_modified = kwargs.get("last_modified", None)
         self.read_only = kwargs.get("read_only", None)
         self.tags = kwargs.get("tags", {})
-        self.value = {"secret_uri": self._secret_uri}
+        self.value = secret_id
+        if not isinstance(secret_id, dict) and isinstance(secret_id, six.string_types):
```

Comment:
?

Reply:
Can remove the second half of this conditional.